tests/unit/module_utils/test_aci.py

```diff
@@ -325,13 +325,13 @@ def test_invalid_response(self):
             return
 
         elif etree.LXML_VERSION < (3, 3, 0, 0):
-            error_text = "Unable to parse output as XML, see 'raw' output. " "Couldn't find end of Start Tag aaa line 1, line 1, column 5"
+            error_text = "Unable to parse output as XML, see 'raw' output. Couldn't find end of Start Tag aaa line 1, line 1, column 5"
         elif PY2:
-            error_text = "Unable to parse output as XML, see 'raw' output. " "Couldn't find end of Start Tag aaa line 1, line 1, column 6 " "(line 1)"
+            error_text = "Unable to parse output as XML, see 'raw' output. Couldn't find end of Start Tag aaa line 1, line 1, column 6 (line 1)"
 
         else:
             error_text = (
-                "Unable to parse output as XML, see 'raw' output. " "Couldn't find end of Start Tag aaa line 1, line 1, column 6 " "(<string>, line 1)"
+                "Unable to parse output as XML, see 'raw' output. Couldn't find end of Start Tag aaa line 1, line 1, column 6 (<string>, line 1)"
```

Comment:
Why does sanity test fail on this line?

Reply:
because of implicit string concatenation